.github/workflows/image-push.yaml

```diff
@@ -0,0 +1,23 @@
+name: Push container image
+on:
+  push:
+    branches:
+      - master
+jobs:
+  build:
+    if: github.repository == 'kubernetes-csi/csi-driver-smb'
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v1
+      - name: Setup Docker buildx
+        id: buildx
+        uses: crazy-max/ghaction-docker-buildx@v3
+        with:
+          buildx-version: latest
+          qemu-version: latest
+      - name: Build and push ARM image
+        run: |
+          docker buildx build \
+          docker push $REPO:$TAG -t $REPO:$TAG \
+            --platform=linux/arm64,linux/amd64 \
+            --output="type=image" .
```

Comment:
@andyzhangx We need to set the registry details here. Could you share information regarding the registry / credentials? Thanks

Reply:
`make smb-container` is here:
https://github.com/kubernetes-csi/csi-driver-smb/blob/e9c2dd3a6ad5d62670177a16cad9b37f4bdabf98/Makefile#L112-L121

I think you need to add one more command to build ARM64 container?